src/bioregistry/curation/literature.py

```diff
@@ -37,3 +37,5 @@ class CurationRelevance(str, enum.Enum):
     unclear = enum.auto()
     #: Completely unrelated information
     irrelevant_other = enum.auto()
+    #: Relevant for training purposes, but not curated in Bioregistry due to poor/unknown quality
+    not_notable = enum.auto()
```

Comment:
Is this the only place we need to add this? I seem to recall also having this info duplicated in the documentation?

Reply:
Yes, it also needs to be updated in https://github.com/biopragmatics/bioregistry/pull/1195. I will update that PR separately

Reply:
Okay, let's merge this first then and also add it on the other PR